src/pages/Profile.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Dimmer, Loader } from 'semantic-ui-react';
+
+// components
+import Navbar from '../components/presentationals/Navbar/Navbar';
+import ProfileSidebar from '../components/containers/ProfileSidebar/ProfileSidebar';
+import ProfileMain from '../components/containers/ProfileMain/ProfileMain';
+
+// styles
+import './Profile.scss';
+
+//actions
+import { getUserProfileRequest } from '../store/modules/profile';
+
+class Profile extends Component {
+  componentDidMount() {
+    this.props.getUserProfileRequest();
+  }
+  render() {
+    const {
+      profile: { data },
+      isLoading,
+    } = this.props;
+    return (
+      <div>
+        <Navbar
+          props={{
+            url:
+              data.image ||
+              'https://res.cloudinary.com/nedy123/image/upload/v1557149386/avatar_pupqrh.svg',
+            class: '',
+          }}
+        />
+        <div className="profile-container">
+          {isLoading && (
+            <Dimmer active>
+              <Loader />
+            </Dimmer>
+          )}
+
+          <ProfileSidebar profile={data} />
+          <ProfileMain profile={data} />
+        </div>
+      </div>
+    );
+  }
+}
+
+Profile.propTypes = {
+  getUserProfileRequest: PropTypes.func,
+  profile: PropTypes.shape({
+    data: PropTypes.object,
+  }),
+  isLoading: PropTypes.bool,
+};
+
+const mapStateToProps = state => {
```

Comment:
This can be rewritten as 
```javascript
const mapStateToProps = state => ({
    profile: state.profileReducer.profile,
    isLoading: state.profileReducer.isLoading,
});

Reply:
This has been resolved to make use of the implicit return